cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/network/model/v7/MessagesApiModel.kt

```diff
@@ -0,0 +1,69 @@
+package com.sourcepoint.cmplibrary.data.network.model.v7
+
+import com.sourcepoint.cmplibrary.data.network.util.Env
+import com.sourcepoint.cmplibrary.model.exposed.CcpaStatus
+import com.sourcepoint.cmplibrary.model.exposed.GDPRPurposeGrants
+import com.sourcepoint.cmplibrary.model.exposed.MessageSubCategory
+import okhttp3.HttpUrl
+import org.json.JSONObject
+
+internal data class MessagesResp(
+    val thisContent: JSONObject,
+    val propertyId: Int,
+    val campaigns: List<MessagesCampaign> = emptyList(),
+    val localState: JSONObject?
+)
+
+internal abstract class MessagesCampaign {
+    abstract val thisContent: JSONObject
+    abstract val type: String
+    abstract val dateCreated: String?
+    abstract val url: HttpUrl?
+    abstract val message: JSONObject?
+    abstract val messageMetaData: JSONObject?
+    abstract val messageSubCategory: MessageSubCategory
+}
+
+internal data class GdprMess(
```

Comment:
Do you mind renaming it to `GdprMessage` instead of "mess" 😅 (I understand it's just me being picky). Same for `CcpaMess`.